frontend/src/components/formDetails/course/CourseForm.tsx

```diff
@@ -60,51 +48,19 @@ const CourseForm = ({
         value={course_description}
         onChange={(e) => updateFields({ course_description: e.target.value })}
       />
+      <label>Price</label>
+      <input
+        required
+        type='number'
+        value={course_price}
+        onChange={(e) => updateFields({ course_price: e.target.valueAsNumber })}
+      />
       <label>category</label>
       <select value={course_category} onChange={handleCategoryChange}>
         <option value='personal'>Personal</option>
         <option value='finance'> Finanace</option>
         <option value='profesional'>Profesional</option>
```

Comment:
Another one here

Reply:
Apologies for the oversight. My form has been replaced with some other values based on the requirement.

I will change them soon.